src/style.css

```diff
@@ -0,0 +1,25 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: 16px;
```

Comment:
remove this

Reply:
I'm removed 15 'background-size: 16px'. Thank you!